src/Bicep.Core/Analyzers/Linter/LinterAnalyzer.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is running really, really slow in vscode.  Several seconds between each pass of the linter.

Reply:
Fixed up configuration logic and reloading is now implemented as well. Speed seems to be dramatically improved.